### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by davidkelly**
Taking 2 locks at the same time in multiple spots in code always seems to lead to deadlocks.  Eventually you edit one of the places, unintentionally invert the order, and there you go.  So, taking these locks one-at-a-time now.

Test Plan:
Smoke test PAL - specifically ignore/unignore.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 069c85c7c250773f1302870d3679e1d1fd187c62 available at: https://gist.github.com/fa53250c4ee5a9883decd70d6bf473b6<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9601-069c85c.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9601-069c85c.dmg

File: libraries/networking/src/NodeList.cpp [hunk 1]
```diff
@@ -850,10 +860,13 @@ void NodeList::ignoreNodeBySessionID(const QUuid& nodeID, bool ignoreEnabled) {
...
     if (!nodeID.isNull() && _sessionUUID != nodeID) {
-        QWriteLocker ignoredSetLocker{ &_ignoredSetLock };
-        QWriteLocker personalMutedSetLocker{ &_personalMutedSetLock };
+        _ignoredSetLock.lockForWrite();
         _ignoredNodeIDs.unsafe_erase(nodeID);
+        _ignoredSetLock.unlock();
+        
+        _personalMutedSetLock.lockForWrite();
         _personalMutedNodeIDs.unsafe_erase(nodeID);
+        _personalMutedSetLock.unlock();
```
Comment (howard-stearns) [inline libraries/networking/src/NodeList.cpp]: whitespace got messed up above. Thanks GitHub. But you get the idea..

Comment (davidkelly) [inline libraries/networking/src/NodeList.cpp]: Ha -  is my previous commit

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8612302f07798cc5cc25cdad738b652d61043635 available at: https://gist.github.com/fa53250c4ee5a9883decd70d6bf473b6<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9601-8612302.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9601-8612302.dmg

**Comment by nbq** [comment]
testing this one

**Comment by nbq** [comment]
Tester: Coal
System: i7 6700k, GeForce 970, Windows 10

* I ran two interfaces
* I smoke tested PAL by muting, ignoring and silencing the other Interface.
* I noticed that on the ignore/unignore there was a slight pause between the avatar disappearing and then showing back up when un-ignored. This still worked exactly as I would expect it to

**PASS**

**Pull request merged**